backend/src/main/java/gov/cdc/usds/simplereport/db/repository/FacilityRepository.java

```diff
@@ -42,4 +43,7 @@ public interface FacilityRepository extends EternalAuditedEntityRepository<Facil
       EternalAuditedEntityRepository.BASE_QUERY
           + " and e.organization = :org order by e.facilityName")
   List<Facility> findByOrganizationOrderByFacilityName(Organization org);
+
+  @Query("from Facility e where :deviceType member of e.configuredDeviceTypes")
```

Comment:
Neat, I didn't know `@Query` could do this.

Reply:
yeah I honestly didn't know either! took me some googling